api/docker/app/Dockerfile

```diff
@@ -9,11 +9,14 @@ RUN composer check-platform-reqs
 COPY api/app/app app
 COPY api/app/config config
 COPY api/app/src src
-RUN composer run-script post-install-cmd --no-interaction
 RUN composer dump-autoload --optimize
 
+
 FROM php:8.1.28-fpm-alpine3.19 AS base
+
 ARG PLATFORM=${PLATFORM:-amd64}
+ARG ENVIRONMENT=${ENVIRONMENT:-production}
```

Comment:
production means more not local image (as in production ready image). It could be any value really as the checks are for local.. don't know about the naming.. I know we use production for other non local image related stuff but maybe it's not super clear that it's not an actual environment.